web/src/compose_state.js

```diff
@@ -68,14 +68,11 @@ function get_or_set(fieldname, keep_leading_whitespace, no_trim) {
 }
 
 export function stream_name() {
-    return compose_recipient.compose_recipient_widget.value();
+    return compose_recipient.selected_stream_name;
```

Comment:
If we do a stream ID value, this can just look up the stream name via `stream_data.js`.

Reply:
Yeah.